trojsten/special/plugin_prask_7_1_1/contants.py

```diff
@@ -0,0 +1,41 @@
+class Level:
+    def __init__(self, url, source_fname, hint):
+        self.url = url
+        self.source_fname = source_fname
+        self.hint = hint
+
+
+LEVELS = [
+    Level("letsplay", "pexeso.html", "Je to skoro ako pexeso."),
+    Level("level2", "urlchange.html", "Pozri sa na URL aktuálnej stránky :)"),
+    Level("level3", "insource.html", "Ak heslo nevidno na stránke, mohlo sa zašiť niekam do kódu."),
+    Level("malina", "morse.html", "Jeden pes je dlhý, druhý zase gulatý. Nie sú to Morseho psi?"),
+    Level(
+        "woof",
+        "button.html",
+        (
+            "Prefix ukazoval, že cez Developer Tools (F12) môžeme meniť stránku. Skús tak spraviť "
+            "tlačidlo stlačitelné. Anglicky slovo 'disabled' znamená 'zakázaný' :P"
+        ),
+    ),
+    Level(
+        "slamastika",
+        "validator.html",
+        "Čo to po tebe len ten validátor chce? Nie je niekde kód, podľa ktorého validuje?",
+    ),
+    Level(
+        "m3dv3d",
+        "cookie.html",
+        "Na obrázku je sušienka, anglicky cookie. Tie tušim Prefix spomínal vo videu.",
```

Comment:
```suggestion
        "Na obrázku je sušienka, anglicky \"cookie\". Tie tušim Prefix spomínal vo videu.",
```

Reply:
`Tie tušim Prefix spomínal vo videu` mi pride ako TODO koment pre teba nez message pre ucastnika :D

Mozno `Kiezby len ten prefix vysvetlil cookies vo videu...`.

Reply:
navyse to ma byt `tuším` (dlhe i)

Reply:
Podľa mňa je `Tie tušim Prefix spomínal vo videu` ok, dal som tam ale aspoň emoji a ofixoval dĺžeň :)